app/gzac/build.gradle

```diff
@@ -18,6 +19,7 @@ dependencies {
     implementation(project(":connector"))
     implementation(project(":objects-api"))
     implementation(project(":besluit"))
+    implementation(project(":haalcentraal"))
```

Comment:
1 te veel

Reply:
Dat is denk ik het nadeel van Code with me 😁